internal/policy/container/has_modified_files.go

```diff
@@ -60,36 +60,69 @@ type packageFilesRef struct {
 // Validate runs the check of whether any Red Hat files were modified
 func (p *HasModifiedFilesCheck) Validate(ctx context.Context, imgRef image.ImageReference) (bool, error) {
 	fs := afero.NewOsFs()
-	layerIDs, packageFiles, packageDist, err := p.gatherDataToValidate(ctx, imgRef, fs)
+	layerIDs, packageFiles, err := p.gatherDataToValidate(ctx, imgRef, fs)
+	if err != nil {
+		return false, fmt.Errorf("could not generate modified files list: %v", err)
+	}
+
+	packageDist, err := p.parsePackageDist(ctx, imgRef.ImageFSPath, fs)
 	if err != nil {
 		return false, fmt.Errorf("could not generate modified files list: %v", err)
```

Comment:
we probably want to update this error correct? so that both errors have different texts to distinguish between the two?

Reply:
They will have different text because the wrapped error will differ.

Reply:
I get that, but wouldn't different leading text be clearer to debug?

Reply:
I don’t believe so. The logic was extracted from the function that would have returned the previous error. Verbatim. In effect, this should be no more or less clear.